src/pages/UsersPage.js

```diff
@@ -36,11 +36,14 @@ export default function UsersPage({ users, goToCreatepage }) {
 }
 
 const Wrapper = styled.section`
-  display: flex;
-  flex-direction: column;
-  gap: 10px;
   list-style-type: none;
-  max-width: min-content;
+  height: 100vh;
+  align-self: center;
+  background: var(--background_dark);
+  display: grid;
+  grid-auto-rows: repeat;
+  overflow-y: scroll;
+  margin-bottom: 20px;
 `
 const SingleEntry = styled.ul`
```

Comment:
single entry is a unordered list but the user entry inside is not a list item

Reply:
Maybe it could help to have a container around all the entries on the user page so that it can be styled better. At the moment the structure is: 
<UserPage>
<Header/>
<ul/>
<ul/>
</UserPage>

I would change the tag from ul to div or section and add a container around them.
do you know what I mean?